soc/sifive/sifive_freedom/fu500/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_SIFIVE_FREEDOM_U500
+	bool
+	select SOC_FAMILY_SIFIVE_FREEDOM
+
+config SOC_SERIES
+	default "fu500" if SOC_SERIES_SIFIVE_FREEDOM_U500
```

Comment:
Shouldn't this be U500? Or shouldn't the Kconfig be `SOC_SERIES_SIFIVE_FREEDOM_FU500`?